sherpa/astro/datastack/ds.py

```diff
@@ -28,6 +27,10 @@
 
 logger = config_logger(__name__)
 
+try:
+    import stk
+except:
+    logger.warning("could not import stk library. CIAO stack files and syntax will be disabled")
```

Comment:
I noted that there looked to be similar-ish logic in the `sherpa/astro/datastack/utils.py` module. I didn't look closely enough to see if the code was close enough to be able to remove one of them. That is, set `Datastack.load_pha` to `load_wrapper(ui.load_pha)`.

Perhaps there's something obviously different that I'm just missing (as I'm only skimming through the code).